### PR TITLE
add daily calculation for specific pool data

### PR DESCRIPTION
**Pull request by jennypollack**
add daily pool data schema
https://thegraph.com/explorer/subgraph/jennypollack/tinlake-rewards?query=dailyPoolData

### CODE REVIEW

File: src/mapping.ts [hunk 1]
```diff
@@ -59,6 +63,66 @@ export function handleCreateProxy(event: Created): void {
...
+    createPool(poolMeta.id.toString())
+    pool = Pool.load(poolMeta.id)
+  }
+
+  log.debug("successfully using this for pool meta id: {}", [poolMeta.id.toString()])
+  return <Pool>pool    
+}
+
+function createYesterdaySnapshot(date: BigInt, block: EthereumBlock): void {
+  let yesterdayTimeStamp = date.minus(BigInt.fromI32(86400))
```
Comment (offerijns) [inline src/mapping.ts:80]: I think it's better to explicitly declare variables like this, e.g. `let dayInSeconds = 86400; ` or add a comment to the line explaining that it's one day.

File: src/mapping.ts [hunk 2]
```diff
@@ -59,6 +63,66 @@ export function handleCreateProxy(event: Created): void {
...
+
+    dailyPoolData.totalDebt = pool.totalDebt
+    dailyPoolData.seniorDebt = pool.seniorDebt
+    dailyPoolData.currentJuniorRatio = pool.currentJuniorRatio
+    dailyPoolData.save()
+  }
+}
+
+// capture a snapshot of the investments by day
+function runRewardLogic(block: EthereumBlock): void {
```
Comment (offerijns) [inline src/mapping.ts:113]: This isn't really related to the rewards right? Maybe this function should have a different name, e.g. `createDailySnapshot()`?

**Review by offerijns — APPROVED** [review]
Looks good! Just left two minor comments.

File: src/mapping.ts (hunk 1 above)
Comment (jennypollack) [reply src/mapping.ts:80]: good call

**Review by offerijns — APPROVED** [review]
LGTM 👍

**Pull request merged**